src/main/java/uk/gov/pay/directdebit/mandate/model/MandateType.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.pay.directdebit.mandate.model;
+
+import org.slf4j.Logger;
+import uk.gov.pay.directdebit.app.logger.PayLoggerFactory;
+
+public enum MandateType  {
+    ONE_OFF("one off"), 
```

Comment:
Should we use `one_off` and `on_demand` as values (without spaces)?

Reply:
👍